code/game/objects/items/devices/scanners.dm

```diff
@@ -165,15 +165,9 @@ REAGENT SCANNER
 		var/mob/living/carbon/human/H = M
 		var/list/damaged = H.get_damaged_organs(1,1)
 		user.show_message("<span class='notice'>Localized Damage, Brute/Burn:</span>",1)
-		if(length(damaged)>0)

```

Comment:
Don't get me started on health analyzer code.

The use of `show_message` over `to_chat`, and how everything is laid out...it's disgusting.